### DISCUSSION TITLE
Differences between alpine and non-alpine based ISO images

### DISCUSSION THREAD

**Discussion by cristian98149** [kairos-io/kairos#1171] [Q&A] [answered]
Hi,
what are the differences between alpine and non-alpine based ISO images?

For example between kairos-alpine-ubuntu-v1.6.0.iso and kairos-ubuntu-v1.6.0.iso?

Thank you

**Comment by mauromorales** [comment, marked as answer]
Alpine-based images need an additional kernel, while other images don't. We currently provide a version with Ubuntu and one with openSUSE.

The reason behind this (pls anyone correct me if I'm wrong) is that Alpine is missing some dependencies to be able to build the intitramfs image properly, hence the additional kernel to do this.

**Reply by Itxaka** [reply]
IIRC its to provide the initrd as currently alpine does not support dracut/mkinitrd which we relay on to work (mounting stuff, kcrypt and so on). So providing the initrd we need to provide the kernel as well so they match.
